datastore/google/cloud/datastore/client.py

```diff
@@ -134,7 +134,7 @@ def _extended_lookup(datastore_api, project, key_pbs,
         loop_num += 1
         lookup_response = datastore_api.lookup(
             project,
-            key_pbs,
+            keys=key_pbs,
```

Comment:
Please don't modify this. I intentionally changed it so that positional arguments would be passed as positional and keyword as keyword. (Also this is unrelated to your PR.)

Reply:
I get errors relating to the tests if I don't change it though.  Should I modify the tests instead?

Reply:
That's because you've edited the `gapic` generated file. Please revert that edit (see my comment there).

Reply:
The errors are this:
https://gist.github.com/chemelnucfin/eee00b41ba2ac8b8f10e1d68bb3047f6
and this:
https://gist.github.com/chemelnucfin/4b010867914d44473b2994b7b8cd6317

Reply:
That's because of your edit in the `gapic` generated file.